sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -180,7 +180,7 @@ public DirectConnectionConfig setIdleEndpointTimeout(Duration idleEndpointTimeou
      * Gets the max connections per endpoint
      * This represents the size of connection pool for a specific endpoint
      *
-     * Default value is 30
+     * Default value is 130
```

Comment:
Can we use the variable in the doc ?

Reply:
```suggestion
     * Default value is {@value #DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT}
```

Reply:
Tried this, but didn't work. 
Tried this as well, didn't work either: 
```
Default value is {@value com.azure.cosmos.DirectConnectionConfig#DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT}
```